src/with-unique.js

```diff
@@ -0,0 +1,56 @@
+import { customElements, dashCase, HTMLElement, keys, sym } from './util';
```

Comment:
All this code because we can't access the constructor in an ES5 way before calling `super()`. `new.target` works, but that only works in ES2015. All we need to do is `this.constructor`, but nooooo.